src/hotspot/share/gc/x/xUnload.cpp

```diff
@@ -75,8 +75,7 @@ class XIsUnloadingOopClosure : public OopClosure {
 
 class XIsUnloadingBehaviour : public IsUnloadingBehaviour {
 public:
-  virtual bool has_dead_oop(CompiledMethod* method) const {
-    nmethod* const nm = method->as_nmethod();
+  virtual bool has_dead_oop(nmethod* const nm) const {
```

Comment:
`nmethod* const nm` => `nmethod* nm`. (ZGC's style is to use const for local variables, but not for variables in the parameter list). The same applies to the rest of the changes to this file.

Reply:
Okay. I did not know that it is only used for locals. I will update code.